docs/developers/blobstream.md

```diff
@@ -28,4 +28,5 @@ Integrating your L2 with Blobstream requires two components: your onchain smart
 You can interact with the Blobstream contracts today on testnet. The Blobstream Solidity
 smart contracts are currently deployed on the following Ethereum testnet:
 
-- [Sepolia](https://sepolia.etherscan.io/address/0xf148a9a767f19edcf9a7d125fe143db1b5792ad6) - Which attests to the data published on the Arabica Devnet
+- [Blobstream on Sepolia](https://sepolia.etherscan.io/address/0xf148a9a767f19edcf9a7d125fe143db1b5792ad6) - Which attests to the data published on the Arabica Devnet
+- [Blobstream X on Goerli](https://goerli.etherscan.io/address/0x67ea962864cdad3f2202118dc6f65ff510f7bb4d)
```

Comment:
to which Celestia testnet does this attest data to?

Reply:
I think this is to Mocha, can you confirm @SweeXordious?

Reply:
We're still asking for a confirmation from succinct

Reply:
https://github.com/celestiaorg/docs/pull/1165/commits/aaf0a03eb2b726af2b3186c68bc1f1a2163dfc8e